openmetadata-spec/src/main/resources/json/schema/tests/testCase.json

```diff
@@ -89,6 +89,10 @@
       "$ref": "../type/entityReferenceList.json",
       "default": null
     },
+    "createdBy": {
```

Comment:
does it require any migration for existing tests?

Reply:
the use-case I am thinking is , if there are existing customers with lot of teset cass and then later adopted the AI recommenation , we should be able to showcase how many are created by users vs AI. I think if we don’t have it populated for users this might not look right. wdyt?

Reply:
Makes sense. Let me add this migration then. I'll use the updatedby field as the ref.